### DISCUSSION TITLE
Signal missed always 30 minutes or more

### DISCUSSION THREAD

**Discussion by jim201** [NightscoutFoundation/xDrip#3363] [Q&A] [answered]
Hi all,
xDrip always seems to display information 30 minutes behind the actual data.  The current data displayed is correct, but it thinks it was data from 30 minutes ago.  Any ideas as to why this is happening? Is this why the reading is presented with the line though it?
Thanks,
Jim
![Screenshot_20240226_102939_xDrip+2](https://github.com/NightscoutFoundation/xDrip/assets/21213218/d742873b-e2d0-489a-aa2f-026abdf355ae)

**Comment by Navid200** [comment]
What is the hardware data source set to?

**Reply by jim201** [reply]
Hi and apologies, it is the carelink follower.

**Comment by Navid200** [comment]
@benceszasz

**Comment by benceszasz** [comment]
@jim201 In which country are you using it?

**Reply by jim201** [reply]
In Australia. I'm in Adelaide which is a funny time zone that is 1/2 hour behind the east coast and 2 1/2 hours ahead of the West Coast, not sure if that makes any difference.

**Reply by benceszasz** [reply, marked as answer]
The 1/2 hour timezone is the root cause of this issue. In the background CareLink follower corrects time data received from CareLink, because the timezone information is missing in some cases (depends on device (7xx, standalone CGM), phone (Android or IOS) and region (EU/US)). For timezone calculation CareLink follower uses an hour based logic, which in this case is wrong, it should be rewritten to a half hour logic.

**Reply by jim201** [reply]
Thanks, I figured as much.
